cdc/owner.go

```diff
@@ -750,6 +754,9 @@ func (o *Owner) flushChangeFeedInfos(ctx context.Context) error {
 		}
 	}
 	if time.Since(o.gcSafepointLastUpdate) > GCSafepointUpdateInterval {
+		if (time.Now().Unix() - int64(minCheckpointTs)) > o.gcBLT {
```

Comment:
what happens when the checkpoint delay exceeds `gcBLT` in a running changefeed?

Reply:
Note that you need to fail any running changefeed whose checkpoint is behind your newly generated `minCheckpointTs`, before you can safely update the safepoint in PD. Otherwise there's a chance that data loss will occur.